lesson16.md

```diff
@@ -0,0 +1,87 @@
+<!-- .slide: id="lesson16" -->
+
+# Basic Frontend - Spring 2023
+
+Lesson 16, Tuesday, 2023-05-23
+
+---
+
+Today's lesson will be the last lesson before starting with the project, and we will be doing a small project to get you up and running!
+
+
+
+The structure is very loose, it is up to you to explore and find what you enjoy!
+
+
+---
+
+# Science, Rockets, and Space!
+
+
+
+---
+
+
+## Spaceflight News API
+
+https://spaceflightnewsapi.net/
+
+https://api.spaceflightnewsapi.net/v4/docs/
+
+---
+
+### Task
+
+Create a simple webpage, that fetches articles or blogs from the API, and show the title, summary, and image.
+
+
+---
+
+Let's add some small touches to the website!
+
+
+---
+
+## SimpleParallax
+
+
+https://simpleparallax.com/
+
+
+https://github.com/geosigno/simpleParallax.js
+
+
+---
+
+### Task
+
+Can you add parallax to your images?
+
+
+---
+
+
+### More stuff to do!
+
+You don't have to do all of this, these are just ideas!
+
+
+
+* allow the users to open the article (with a link)
+* add `next` and `previous` pages buttons
+* Show loading while the data is being fetched.
+* Show the news site
+* Show the articles creation or publishing date.
+* Allow the user to search for keywords
+  * In the title
+  * Or summary
+  * Or both!
+* Or show article earlier or later than a certain date
```

Comment:
Does all this text fit on a single slide? :)

Reply:
I think it does, barely.